release notes/v0.48.0.md

```diff
@@ -33,8 +33,7 @@ _Format as `<number> <present_verb> <object>. <credit>`_:
 
 ## Bug fixes
 
-_Format as `<number> <present_verb> <object>. <credit>`_:
-- _`#111` fixes race condition in runtime_
+- [browser#1042](https://github.com/grafana/xk6-browser/pull/1042), [browser#1069](https://github.com/grafana/xk6-browser/pull/1069) Fixes `browserContext.waitForEvent`. This fix involved promisifying the `waitForEvent` API as well as various internal refactors.
```

Comment:
My understanding is that https://github.com/grafana/xk6-browser/pull/1069 is not specifically related with `waitForEvent`, as it also refactors the usage of `TaskQueue` for `Page.on`. Should we remove it from here and instead add an entry for it under `Maintenance and internal improvements` section?

Reply:
Yeah, good point, it does include quite a bit of refactoring to move the taskqueue out of the common package and into the mapping layer. The main motivation though was due to the issue of not working with the `taskqueue` when using the predicate function for `waitForEvent`, i.e. without this change `waitForEvent` wouldn't work. Happy to move it to `Maintenance and internal improvements` though if you still think it should, WDYT?

Reply:
Ok, happy with that change added it to a fixup: ef9213368a9a5b7010584fd532d97c329a40b791

Reply:
How specific should we be with these internal refactors? To be honest I'm never sure of what should be the level of detail in these cases. Considering that this is a completely internal change, I don't disagree with the current text, but I wonder if it should be more specific such as _"Internal refactor to move Goja's TaskQueue away from core implementation"_ (or similar).

Since you are a reviewer for this also, any thoughts @oleiade ?

Reply:
Mind if i merge this and we can open a new issue for this if we need?